book/10-git-internals/sections/objects.asc

```diff
@@ -139,6 +139,11 @@ $ git cat-file -p master^{tree}
 The `master^{tree}` syntax specifies the tree object that is pointed to by the last commit on your `master` branch.
 Notice that the `lib` subdirectory isn't a blob but a pointer to another tree:
 
+[NOTE]
+====
+Depending on what shell you use, you may encounter errors when using the `master^{tree}` syntax. In PowerShell or CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p mater^^{tree}`. If you're using ZSH, the `^` character is used for globbing, so you have to enclose the whole expression in quotes: `git cat-file -p "master^{tree}"`
```

Comment:
Sorry, one more request: could you reformat this with one sentence on each source-code line? It makes reviewing PRs _much_ easier in the future.

Reply:
sure no problem